extensions/ql-vscode/src/remote-queries/analyses-results-manager.ts

```diff
@@ -145,7 +148,11 @@ export class AnalysesResultsManager {
     return queryResults;
   }
 
-  private isAnalysisDownloaded(analysis: AnalysisSummary): boolean {
+  private isAnalysisInMemory(analysis: AnalysisSummary): boolean {
     return (this.analysesResults.get(analysis.downloadLink.queryId) || []).some(x => x.nwo === analysis.nwo);
   }
+
+  async isAnalysisDownloaded(analysis: AnalysisSummary): Promise<boolean> {
```

Comment:
Where is this used? Should it be async?

Reply:
This is not used (yet). Once we start trying to proactively determine what has already been downloaded, we're going to need this function. I had started trying to do this, but gave up when I realized it is harder than I thought. I'll work on this part later. And will remove this function for now.